projects/ccf-rui/src/app/modules/content/content.component.ts

```diff
@@ -24,7 +25,9 @@ export class ContentComponent {
    *
    * @param model The model state
    */
-  constructor(readonly model: ModelState) {}
+  constructor(readonly model: ModelState, readonly page: PageState) {
+    page.tutorialMode$.subscribe();
```

Comment:
@axdanbol is this kosher? Not sure I've seen state having to have a subscribe like this.

Reply:
No, it's not. I don't know why this would even be needed.

Reply:
I was going off of the ccf-eu app.component.ts constructor when writing this.  If it's not the best way I'll change it.

` constructor(readonly data: DataState, readonly dataSourceService: DataSourceService, readonly theming: ThemingService) {
    data.listData$.subscribe();
    data.aggregateData$.subscribe();
    data.termOccurencesData$.subscribe();
    this.ontologyTerms$ = data.filter$.pipe(pluck('ontologyTerms'));
  }`

Reply:
That state works differently than normal (it keeps track of when queries are running + I never got around to fixing it). For PageState you don't need any of that just use the `tutorialMode$` variable normally

Reply:
Okay, sounds good.  I'll get those changed.